packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -15,6 +15,7 @@ significantly improves the user experience when using DevTools embedded
 an IDE. (#7030)[https://github.com/flutter/devtools/pull/7030]
 * Added support for filtering with regular expressions in the Logging, Network, and CPU profiler
 pages - (#7027)[https://github.com/flutter/devtools/pull/7027]
+* Update `package:vm_service` constraint to `^14.0.0`. - [#6953](https://github.com/flutter/devtools/pull/6953)
```

Comment:
I think we'll also want to explicitly mention that this means that older Dart + Flutter applications won't be compatible with the updated network profiler.

Reply:
Added a comment.

Reply:
We shouldn't need to comment on backwards compatibility because it is assumed DevTools only works with the Dart SDK it is provided in.

Reply:
I've removed the bullet point I added.